app/src/main/java/com/hedvig/app/util/extensions/ContextExtensions.kt

```diff
@@ -141,6 +142,18 @@ fun Context.showAlert(
         }
         .show()
         .apply {
+            getButton(AlertDialog.BUTTON_POSITIVE).apply {
```

Comment:
This is probably not a good way to theme an AlertDialog 😞

I think that by default AlertDialogs use the theme provided in the attribute `?alertDialogTheme`, check that out!